help/pt_BR/config.yml

```diff
@@ -1,61 +1,74 @@
 
-        gui_render_mouse: Renderizar o cursor do mouse nas GUIs
-
-        minecrafttp_teleport: Usar /minecraft:tp
-        tp_teleport: Usar /tp
-        aspect_ratio: Proporção da tela
-
-    outside: 
+    outside:
         title: Modo externo
         tooltip: Opções de configuração relacionadas ao modo externo
         
-        camera_outside: Modo externo
-        camera_outside_hide_player: Ocultar jogador
-        camera_outside_sky: Consertar céu
-        camera_outside_hide_guis: Esconder GUIs
-        camera_outside_hide_overlays: Ocultar sobreposições
+        enabled: Modo externo
+        hide_player: Ocultar jogador
+        sky: Fixar céu
 
-    overlay:
-        title: Sobreposição do editor de câmera
-        tooltip: Permite configurar uma imagem de sobreposição que seria usada no editor de câmera
+    editor:
+        title: Editor de câmera
+        tooltip: Opções de configuração usadas no editor de câmera
         
-        camera_editor_overlay: Ativar sobreposição
-        camera_editor_overlay_rl: Caminho do recurso para sobrepor
-        camera_editor_f1_tooltip: F1 ocultar dica
+        sync: Modo de sincronização
+        loop: Loop do suporte
+        overlay: Ativar sobreposição
+        overlay_rl: Caminho de sobreposição
+        f1_tooltip: F1 ocultar dica
+        position: Exibir posição
+        rule_of_thirds: Regra dos terços
+        crosshair: Mira
+        letter_box: Caixa postal
+        aspect_ratio: Proporção da tela
+
+    flight:
+        title: Modo de voo
+        tooltip: Esta categoria permite que você defina teclas de atalho para as teclas do modo de voo no editor de câmera
+
+        forward: Mover para Frente
+        backward: Mover para Trás
+        left: Mover para Esquerda
+        right: Mover para Direita
+        up: Mover para Cima
+        down: Mover para Baixo
+        camera_up: Inclinar para Cima
+        camera_down: Inclinar para Baixo
+        camera_left: Deslocar para Esquerda
+        camera_right: Deslocar para Direita
+        fov_minus: Diminuir FOV
+        fov_plus: Aumentar FOV
+        roll_minus: Rolar para Esquerda
+        roll_plus: Rolar para Direita
+        speed_minus: Diminuir Velocidade
+        speed_plus: Aumentar Velocidade
 
     smooth:
         title: Câmera suave
         tooltip: Opções de câmera suave
 
-        camera_smooth_clamp: Limite de inclinação
-
+        enabled: Câmera suave ativada
+        x_friction: Deslizamento do mouse X
+        y_friction: Deslizamento do mouse Y
+        clamp: Limite de inclinação
         roll_speed: Velocidade de rolagem
-        roll_friction: Atrito de rolagem
-
+        roll_friction: Deslizamento de rolagem
         fov_speed: Velocidade do FOV
-        fov_friction: Atrito do FOV
-
-        smooth_enabled: Câmera suave ativada
-        mouse_x_friction: Atrito do mouse X
-        mouse_y_friction: Atrito do mouse Y
+        fov_friction: Deslizamento do FOV
```

Comment:
You missed here `minema` [section](https://github.com/mchorse/aperture/compare/1.3.6...1.12#diff-7c88845ae8f8e0f54123ce9e3dfaf823R76).